### PR TITLE
Upload Features [CHA-5196]

### PR DESCRIPTION
**Pull request by rooftoofwoof**
Add a method to upload features to ChalkClient.

### CODE REVIEW

**Review by melrchen — APPROVED** [review]
thank you!

**Comment by rooftoofwoof** [comment]
The test that's failing is the integration test against staging for the gRPC flavor of upload features.  Once Raphael merges that into dev, re-running the test should give us a pass.

File: src/main/java/ai/chalk/client/BuilderImpl.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package ai.chalk.client;
 
 import ai.chalk.exceptions.ChalkException;
+import ai.chalk.exceptions.ClientException;
```
Comment (rooftoofwoof) [inline src/main/java/ai/chalk/client/BuilderImpl.java:4]: remove

File: src/main/java/ai/chalk/client/GRPCClient.java [hunk 2]
```diff
@@ -173,7 +171,9 @@ public void printConfig() {
         logger.log(System.Logger.Level.ERROR, "Config printing for GRPC client not yet implemented");
     }
 
-    private RequestHeaderInterceptor getRequestHeaderInterceptor(String environmentId) {
+    private RequestHeaderInterceptor getRequestHeaderInterceptor(
+        @Nullable String environmentId
```
Comment (rooftoofwoof) [inline src/main/java/ai/chalk/client/GRPCClient.java:175]: override

**Pull request merged**